docs/source/azure/policy/resources/publicip.rst

```diff
@@ -19,38 +19,12 @@ Actions
 Example Policies
```

Comment:
Maybe we can add the example of finding VMs with public IPs, because that is a common policy.

Reply:
That is under the VM examples!